### ISSUE TITLE
Outdated dependency: commons-lang3 (3.9 instead of 3.8.1) 

### ISSUE THREAD

**Issue by dzc34** [Asqatasun/Contrast-Finder#213]
## Observed behaviour
outdated dependency : commons-lang3 

## Expected behaviour
-  upgrade  commons-lang3 (3.9 instead of 3.8.1) 
   - Requires Java 8.0 (Tomcat with JDK8)
   - https://commons.apache.org/proper/commons-lang/
   - Release Notes
     - https://commons.apache.org/proper/commons-lang/release-history.html
     - https://commons.apache.org/proper/commons-lang/release-notes/RELEASE-NOTES-3.9.txt

-----------
previous: #180

**Comment by dzc34** [comment]
Fail for Ubuntu 14.04 / Tomcat 7 / JDK7
```
 ./docker/build_and_run-with-docker.sh  -l  --source-dir $(pwd) --docker-dir docker/SNAPSHOT-local/distrib/ubuntu-14.04_tomcat-7 --log-build
docker exec -ti contrast.finder /bin/bash
```

------------------

```
cat /var/log/tomcat7/localhost_access_log.(...).txt
   (...) "GET /contrast-finder/ HTTP/1.1" 500 
   (...) "GET /contrast-finder/ HTTP/1.1" 500 
```
-------------------------

```
cat /var/log/tomcat7/localhost.(...).log
INFO: Initializing Spring FrameworkServlet 'contrast-finder' (...)
SEVERE: Servlet.service() for servlet [contrast-finder] in context with path [/contrast-finder] 
threw exception [Handler dispatch failed; nested exception is java.lang.UnsupportedClassVersionError: 
org/apache/commons/lang3/StringUtils : Unsupported major.minor version 52.0 
(unable to load class org.apache.commons.lang3.StringUtils)] with root cause
java.lang.UnsupportedClassVersionError: org/apache/commons/lang3/StringUtils : 
Unsupported major.minor version 52.0 (unable to load class org.apache.commons.lang3.StringUtils)
```

**Comment by dzc34** [comment]
closed by #259
